src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/PrivilegesInterceptorImpl.java

```diff
@@ -119,14 +119,14 @@ public ReplaceResult replaceKibanaIndex(final ActionRequest request, final Strin
         }
 
         //intercept when requests are not made by the kibana server and if the kibana index/alias (.kibana) is the only index/alias involved
```

Comment:
Not sure if we need to rename kibana to openSearchDashboards in these comments.
All the comments that contain kibana: 121, 145, 155, 230

Reply:
These are indices, which is not in the scope of this PR. We'll change these things when we migrate indices.